lib/hexpm/release_tasks.ex

```diff
@@ -33,34 +33,34 @@ defmodule Hexpm.ReleaseTasks do
     stop()
   end
 
-  def migrate() do
+  def migrate(args \\ []) do
```

Comment:
I was able to run seeds/migration/etc like this:

    bin/hexpm eval 'Hexpm.ReleaseTasks.migrate()'

in order to pass args, however, instead of "--step 1" like before, now we have to do it like this:

    bin/hexpm eval 'Hexpm.ReleaseTasks.migrate(~w(--step 1))

we can probably write some wrapper scripts for it for convenience, not sure.

Reply:
Or using environment variables. We could support argv in eval but  I am not sure if we should, since it wouldn't work for RPC.

Reply:
since we're already evaling, @josevalim suggested we may as well just do: 

    bin/hexpm eval 'Hexpm.ReleaseTasks.migrate(step: 1)

that seems nicer.